contrib/gardener/scripts/deploy-karydia-cp

```diff
@@ -222,7 +222,7 @@ spec:
       automountServiceAccountToken: false
       containers:
       - name: karydia
-        image: karydia/karydia:0.1.0
+        image: eu.gcr.io/gardener-project/karydia/karydia:0.0.1
```

Comment:
current CI/CD pipeline versioning has to be discussed first

Reply:
But the last version was `0.1.0` why should we decrease the version number to `0.0.1`? That doesn't make sense to me.

Reply:
Just had a look at "gcr.io" and it looked like the version format was ```0.0.1```.

I have no preferences for the versioning...

Reply:
We decided to go with just `eu.gcr.io/gardener-project/karydia/karydia` (no version tag) AND change `imagePullPolicy: IfNotPresent` to `imagePullPolicy: Always` for production at the moment. This should be changed later, so, maybe a `TODO` comment would be nice.

Reply:
Done.